docs_src/microservices/device/Ch-DeviceServices.md

```diff
@@ -58,6 +58,18 @@ All device services must perform the following tasks:
 
 As you can imagine, many of these tasks (like registering with core metadata) are generic and the same for all device services and thereby provided by the SDK.  Other tasks (like getting sensor data from the underlying device) are quite specific to the underlying device.  In these cases, the device service SDK provides empty functions for performing the work, but the developer would need to fill in the function code as it relates to the specific device, the communication protocol, device driver, etc.
 
+### Device Service Timestamps
+
+Core metadata tracks two timestamps for each device service:  last connected and last reported.  Both last connected and last reported are saved as int64 values - representing time (Unix time) in milliseconds
+
+#### Last Connected
+
+Last connected indicates the last time a device service was engaged and responded to a request.  The last connected timestamp would be updated when a command request (such as that from core command) is responded to or when the device service responds to a core metadata callback.
```

Comment:
I this doesn't occurs with the 2.x Command Service. 
https://github.com/edgexfoundry/edgex-go/blob/main/internal/core/command/controller/http/command.go#L125
https://github.com/edgexfoundry/edgex-go/blob/main/internal/core/command/controller/http/command.go#L93

@cloudxxx8 , who is actually setting LastConnected?
I see it can be set in a PATCH here, but who makes the request?
https://github.com/edgexfoundry/edgex-go/blob/7c172932d67f5deb00d2ba71bf7bdd749298babf/internal/core/metadata/application/deviceservice.go#L82

Reply:
yes, metadata API can update it, and I thought Device Service should call the API, not core-command, because it includes the metadata callback API calls.